bin/compare.sh

```diff
@@ -16,25 +16,13 @@ export NODE_ENV=test
 helmfile template $templateArgs --output-dir-template="../$targetDirA/{{.Release.Namespace}}-{{.Release.Name }}"
 
 git checkout $compareBranch
+# we remove previously rendered manifests so they are not mixed up with newly rendered
+rm -rf $targetDirB
 helmfile template $templateArgs --output-dir-template="../$targetDirB/{{.Release.Namespace}}-{{.Release.Name}}"
 git checkout $currentBranch
 
-set +e
-diff_output=$(diff -q -r $targetDirA $targetDirB)
-set -e
-# Process each line of diff output
-
-echo "$diff_output" | while read -r line; do
-  # Check if the line indicates a difference
-  if [[ $line == *" and "* ]]; then
-    # Extract the paths using cut
-    first_path=$(echo $line | cut -d' ' -f2)
-    second_path=$(echo $line | cut -d' ' -f4)
-
-    # Use dyff to compare the files
-    dyff between "$first_path" "$second_path"
-  fi
-done
+# order of arguments matters so new chanages are green color
+bin/dyff.sh $targetDirB $targetDirA
```

Comment:
The naming is confusing you put in DirB but in the script is DirA. Would be nice if you could change that so it reads a little bit better.

Reply:
You are right. I will change it.